### ISSUE TITLE
Rancher-Compose: Labels need to allow stackname/servicename as the label for target services in a different stack.

### ISSUE THREAD

**Issue by deniseschannon** [rancher/rancher#1693]
Since we allow cross stack linking, we run into issues when we have services with the same name using rancher-compose. Rancher-Compose needs to support label key in this syntax:

```
 io.rancher.loadbalancer.target.ENVIRONMENT_NAME/SERVICE_NAME
```

Rancher-Compose v0.2.5
Currently, we're only able to creating the same routing rules for the service names that are the same across stacks. 
```
lb:
  image: rancher/load-balancer-service
  ports:
  - 80
  links: 
  - web1:web1
   external_links:
  - differentstack/web1:web1
  labels:
    io.rancher.loadbalancer.target.web1: app.example.com:80/foo=8000
```

We want to be able to support this rancher-compose for cross-stack services of the same name.
```
lb:
  image: rancher/load-balancer-service
  ports:
  - 80
  links: 
  - web1:web1
   external_links:
  - differentstack/web1:web1
  labels:
    io.rancher.loadbalancer.target.web1: app.example.com:80/foo=8000
   # Add support for `otherstack/servicename` in the key 
    io.rancher.loadbalancer.target.differentstack/web1: '90'
```

The fix in Rancher-Compose needs to be released with the fix in Rancher, where Rancher will generate the `docker-compose.yml` file with the same key.

**Comment by deniseschannon** [comment]
Must be released in conjunction with #1694

**Comment by sangeethah** [comment]
With Rancher-Compose v0.2.5 , we are able to do create LB services for services with same name across different stacks.

Following is the example rancher-compose.yml and docker-compoase.yml with which i am able to create LB services for services with same name across different stacks with host name routing:

``
>> docker-compose,yml

lbweb:
  ports:
  - 7878:80
  restart: always
  external_links:
  - teststack2/web1:web1
  - teststack3/web1:web1
  labels:
    io.rancher.loadbalancer.target.teststack3/web1: xyz
    io.rancher.loadbalancer.target.teststack2/web1: abc
  tty: true
  image: rancher/load-balancer-service
  stdin_open: true
>> rancher-compose,yml
lbweb:
  scale: 1
  load_balancer_config:
    name: lbweb config
root@sangee01:/home/sangeethahariharan1/v025/rancher-compose-v0.2.5# 
```
